apt-hook/20apt-esm-hook.conf

```diff
@@ -1,13 +1,5 @@
 APT::Update::Post-Invoke-Stats {
-	"[ ! -f /usr/lib/ubuntu-advantage/apt-esm-hook ] || /usr/lib/ubuntu-advantage/apt-esm-hook post-invoke-stats || true";
-};
-
-APT::Install::Post-Invoke-Success {
-	"[ ! -f /usr/lib/ubuntu-advantage/apt-esm-hook ] || /usr/lib/ubuntu-advantage/apt-esm-hook post-invoke-success || true";
-}; 
-
-APT::Install::Pre-Invoke {
-	"[ ! -f /usr/lib/ubuntu-advantage/apt-esm-hook ] || /usr/lib/ubuntu-advantage/apt-esm-hook pre-invoke || true";
+	"[ ! -f /usr/lib/ubuntu-advantage/apt-esm-hook ] || /usr/lib/ubuntu-advantage/apt-esm-hook || true";
```

Comment:
there is now only one use-case for the hook, which is rendering the message templates with esm package counts/names, so no cli args are required anymore